src/v/pandaproxy/kafka_client_cache.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "pandaproxy/kafka_client_cache.h"
+
+#include "pandaproxy/logger.h"
+
+namespace pandaproxy {
+
+kafka_client_cache::kafka_client_cache(
+  YAML::Node const& cfg,
```

Comment:
We usually use "west const" `const YAML::Node&`.

Reply:
We do? I was following the pattern in kafka::client
```cpp
explicit client(YAML::Node const& cfg);
```

Nevertheless, I'll change.

Reply:
I as meant to delete this comment - we're actually not consistent here. It's fine as-is.